### PR TITLE
provide fallback when API fails to load

### PR DESCRIPTION
**Pull request by zbtirrell**
Also:
* updated API URL to point directly at theeventscalendar.com
* made the 5th product span full width for improved promotion

See: https://central.tri.be/issues/63558

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [app-shop.php:23](https://github.com/moderntribe/tribe-common/blob/83a226f447fb9178f532f58656219d1c69e6c9e6/src/admin-views/app-shop.php#L23)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$product' | [app-shop.php:46](https://github.com/moderntribe/tribe-common/blob/83a226f447fb9178f532f58656219d1c69e6c9e6/src/admin-views/app-shop.php#L46)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$product' | [app-shop.php:49](https://github.com/moderntribe/tribe-common/blob/83a226f447fb9178f532f58656219d1c69e6c9e6/src/admin-views/app-shop.php#L49)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
:bangbang: This Pull Request is against a Central ticket for "NR Maintenance Release". It shouldn't be merged until the Central ticket has been changed to "Current Release" or "CR Maintenance Release".

File: src/Tribe/App_Shop.php [hunk 1]
```diff
@@ -107,27 +107,24 @@ public function enqueue() {
...
+			$banner = null;
+			$categories = null;
+
 			$remote = $this->get_all_products();
 
 			if ( ! empty( $remote ) ) {
-				$products = null;
 				if ( property_exists( $remote, 'data' ) ) {
 					$products = $remote->data;
+					$categories = array_unique( wp_list_pluck( $products, 'category' ) );
```
Comment (bordoni) [inline src/Tribe/App_Shop.php]: can we pass it on a `array_filter` to prevent empty ones?

Comment (zbtirrell) [inline src/Tribe/App_Shop.php]: I just relocated this line, it is otherwise unchanged.

**Comment by bordoni** [comment]
👍

**Pull request merged**